lang/java/avro/src/test/java/org/apache/avro/io/parsing/TestResolvingGrammarGenerator.java

```diff
@@ -83,6 +83,21 @@ public void testRecordMissingRequiredFieldError() throws Exception {
     }
   }
 
+  @Test
+  public void testDifferingEnumNamespaces() throws Exception {
+    Schema enumSchema1 = SchemaBuilder.builder().enumeration("Enum").namespace("namespace1").symbols("Symbol1",
+        "Symbol2");
+    Schema enumSchema2 = SchemaBuilder.builder().enumeration("Enum").namespace("namespace2").symbols("Symbol1",
+        "Symbol2");
```

Comment:
Can we use schema constants from `TestSchemas` here? Then we can do compatibility checks using the same schemas.

Reply:
I added this change in [this commit](https://github.com/apache/avro/pull/1381/commits/795b080e9a76f5095300e7a1bccb0b137bfe0067) - though it required changing two of the schema static members to be public, as they were previously package-protected. I can undo this commit if the small inconsistency in visibility is deemed a bit messy, but IMO it's fairly harmless since they're only test values.